esf_main.tex

```diff
@@ -853,18 +855,34 @@ \subsubsection{Charging}
 	    \centering
 	    \begin{tabular}{|l|l|}
 	        \hline
-	        Charger Type: & TODO \\ \hline
-	        Maximum charging power & TODO W \\ \hline
-	        Maximum charging voltage & TODO V \\ \hline
-	        Maximum charging current & TODO A \\ \hline
-	        Interface with accumulator & TODO \\ \hline
-	        Input voltage & TODO \\ \hline
-	        Input current & TODO \\ \hline
+	        Charger Type: & ELCON PFC 2000+ TCCH-288-6  \\ \hline
+	        Maximum charging power & 24.9 A \\ \hline
+	        Maximum charging voltage & 4.15 V (Per Cell) \\ \hline
+	        Maximum charging current & 6 A \\ \hline
+	        Interface with accumulator & CAN-Bus \\ \hline
```

Comment:
I think this is talking about the high-voltage line, but I could be wrong. @aehoppe or @rileychapman can probably clarify.